### DISCUSSION TITLE
Cheetos Jumbo

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#513] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/cheetos-jumbo/

**Comment by brunoaugusto** [comment]
![null](<https://i.postimg.cc/K81S2913/heavy-breathing-cat.png>)

**Comment by one-eyed-willy** [comment]
Você fazendo referência ao [Super Slam](<https://www.vish.blog.br/videos/super-slam/>). Cê tá bem?

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/cheetos-jumbo/#comment_1087523938>):

> [![null](<https://camo.githubusercontent.com/fab795c0b36fc9a3ba5b09fb2d3fcb3e3f2df56becb76a20f0c8d74d8196ffca/68747470733a2f2f692e706f7374696d672e63632f4b383153323931332f68656176792d627265617468696e672d6361742e706e67>)](<https://camo.githubusercontent.com/fab795c0b36fc9a3ba5b09fb2d3fcb3e3f2df56becb76a20f0c8d74d8196ffca/68747470733a2f2f692e706f7374696d672e63632f4b383153323931332f68656176792d627265617468696e672d6361742e706e67>)

Minha reação exata

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/cheetos-jumbo/#comment_1087527904>):

> Você fazendo referência ao [Super Slam](<https://www.vish.blog.br/videos/super-slam/>). Cê tá bem?

Quem é você e o que fez com Najarin?

**Comment by najarin-the-loremaster** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/curiosidades/cheetos-jumbo/#comment_1087529869>):

> Quem é você e o que fez com Najarin?

Confesso que eu ri mais do que deveria com aquele vídeo
